### PR TITLE
Update what we are tracking on a video ga event

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@alanmoo review?

What this does is:

1. Add a label to ga video related events with the video's title, so we know if certain videos are doing better than others.
2. Track a finer level of video completion, so instead of 75% we now track 25% 50% and 75%.
3. Created a function to handle this so I didn't need to copy and paste fixes all over the place.

File: src/components/encrypt-video.js [hunk 1]
```diff
@@ -9,9 +9,14 @@ module.exports = React.createClass({
   },
   getInitialState() {
     return {
-      seventyFivePercentDone: false
+      nextPercentDone: 0.25
     };
   },
+  videoGaEvent(props) {
+    props.category = props.category || "Video";
+    props.label = props.label || this.context.intl.formatMessage({id: this.props.video.title});
```
Comment (alanmoo) [inline src/components/encrypt-video.js]: What happens if the video title isn't localized? Most likely scenario seems to be title is localized, video isn't, or neither are localized and user in another region is seeing English title, which might affect the en-US stats. Not necessarily a blocker here, just want to point it out.

**Comment by alanmoo** [comment]
Code all looks solid (without having access to GA in the back end to check data). Just the one comment, which you may not need to worry about.

File: src/components/encrypt-video.js (hunk 1 above)
Comment (ScottDowne) [inline src/components/encrypt-video.js]: I thought about this but couldn't find a terribly pressing reason to do anything here.

Generally we havn't been doing tests, campaigns, or looking at data across multiple locales. We tend to look at data based on a campaign, and a campaign tends to target a locale. So a de campaign should not contain metrics from en-US, and we filter this via the url. So as long as the title is consistent for all titles in that url during the campaign, we should be fine.

As to what actually happens, if the locale doesn't exist, the user is directed to the closest match.

If the locale exists, but isn't complete (title isn't translated yet) it stores the id for the strings instead, which, is probably fine too.

Comment (alanmoo) [inline src/components/encrypt-video.js]: Ah, filtering by url takes care of any issues, I was thinking you'd be looking across locales. Sounds good!

**Pull request merged**